Lib/idlelib/README.txt

```diff
@@ -280,3 +280,11 @@ first back-ported feature after 3.n.0 is released. Since each older
 version file gets a different number of backports, it is easiest to
 make a separate PR for each file and label it with the backports
 needed.
+
+Github repository and issues
+------
```

Comment:
The preceding sections use lines with the same char count as the respective heading.

Reply:
Fixed.